components/server/src/server.rs

```diff
@@ -1504,6 +1507,18 @@ fn get_lock_dir() -> String {
     "TIKV_LOCK_FILES".to_owned()
 }
 
+// Make the lock files be accessed by all users. Only support in unix platform.
+#[cfg(unix)]
+fn set_path_all_permission<P: AsRef<Path>>(path: P) {
+    use std::fs::{set_permissions, Permissions};
+    use std::os::unix::fs::PermissionsExt;
+    // this may failed when setting a file created by root.
+    let _ = set_permissions(path, Permissions::from_mode(0o777));
```

Comment:
Should we log this error?

Reply:
If tikv-server started by root, it will return error when starting by normal user. I don't think this behavior is a error?

Reply:
Got it.